packages/vue/src/components/molecules/SfReview/SfReview.html

```diff
@@ -0,0 +1,30 @@
+<section class="sf-review">
+    <!-- @slot Review author. Slot content will replace default <div> tag (bind 'author' string). -->
+    <slot name="author" v-bind="{author}">
+        <div class="sf-review__author">{{ author }}</div>
+    </slot>
+    <!-- @slot Review information. Slot content will replace default <div> tag and its inner <div> tags (bind 'finalRating' computed property- boolean|number, 'finalMaxRating' computed propery- number, 'date' string). -->
+    <slot name="info" v-bind="{finalRating, finalMaxRating, date}">
+        <div class="sf-review__info">
+            <div v-bind:class="{ 'sf-review__rating': finalRating }">
```

Comment:
```suggestion
            <div :class="{ 'sf-review__rating': finalRating }">
```
Please use short syntax